### PR TITLE
Feat: adjust RC validity (EXPOSUREAPP-11814)

### PR DESCRIPTION
**Pull request by mlenkeit**

### CODE REVIEW

File: test/fixtures/ccl/dcc-series-recovery.yaml [hunk 1]
```diff
@@ -81,9 +81,20 @@
         mostRecentVaccination: 
         hasBooster: false
         verificationCertificates:
-          - certificate: rc1    
-    # recovery gets invalid after 91 days
-    - time: rc1+P91D
+          - certificate: rc1
+    # recovery still valid after 180 days
```
Comment (Ein-Tim) [inline test/fixtures/ccl/dcc-series-recovery.yaml:85]: Same question here: Why is this change done?

Comment (mlenkeit) [reply test/fixtures/ccl/dcc-series-recovery.yaml:85]: @Ein-Tim this change is based on the latest discussions with our stakeholders today. This also seems to be in line with reports such as https://www.berliner-zeitung.de/coronavirus/covid-19-genesenen-zertifikate-wieder-sechs-monate-gueltig-li.211569

Comment (Ein-Tim) [reply test/fixtures/ccl/dcc-series-recovery.yaml:85]: Okay, thanks @mlenkeit.

Maybe the field on RCs should be changed to "Maximal gültig bis:". I will raise this in the CovPass repo.

Comment (mlenkeit) [reply test/fixtures/ccl/dcc-series-recovery.yaml:85]: > Okay, thanks @mlenkeit.
> 
> Maybe the field on RCs should be changed to "Maximal gültig bis:". I will raise this in the CovPass repo.

@Ein-Tim the _valid until_ attribute of the recovery certificate is - to my knowledge - not evaluated by any (German) app for checking the validity. They determine the validity based on _date of first positive test_.

**Pull request merged**